gnucash/gnome/dialog-sx-since-last-run.c

```diff
@@ -1187,6 +1187,10 @@ dialog_response_cb (GtkDialog *dialog, gint response_id, GncSxSinceLastRunDialog
     g_list_free (app_dialog->created_txns);
     app_dialog->created_txns = NULL;
 
+    case GTK_RESPONSE_HELP:
+            gnc_gnome_help (GTK_WINDOW(dialog), HF_HELP, HL_SX_SLR);
+            return;
+
```

Comment:
In this position, the help page is opened when you press the OK button as it using falling through to the help case.
So move these lines to be the first case, above the OK case and change the 'return' to be 'break'. Also make sure the alignment is correct, in this case remove 4 spaces from the two lines.
With these changes and a rebase, this should be OK.

Reply:
@Bob-IT For your technical-debt-reduction list, the body of `GTK_RESPONSE_OK` desperately needs to be moved to at least one and maybe two functions.